src/api/auth/authorization.ts

```diff
@@ -1,8 +1,12 @@
 import jwt from 'jwt-simple';
+import { Config } from '../../config/config.js';
 
 const BEARER_TOKEN_PATTERN = /^Bearer [-_=.0-9a-zA-Z]+$/i;
 
-async function getUserInfo(req, config) {
+async function getUserInfo(
+  req: AWSLambda.APIGatewayProxyEvent,
```

Comment:
From slack convo: it doesn't look like we ever import an `AWSLambda` type into this file. How does it know what that type is?

Reply:
Yeah, it's a fair question. After doing a little bit of research, we see that we have the following dependency:

https://github.com/tnc-ca-geo/animl-api/blob/2e973b481f8b5f97378d83940f4d2025dac04088/package.json#L31

That dependency has the following dependency:

https://github.com/apollo-server-integrations/apollo-server-integration-aws-lambda/blob/c5a2c2dbae47b6e0b3852cea82df7799fda51529/package.json#L34

That adds the `index.d.ts` file which adds types to the [global scope](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/global-d-ts.html) (_I think..._). 
 https://github.com/DefinitelyTyped/DefinitelyTyped/blob/22683376440b9f13f263630615ca32e442b610e6/types/aws-lambda/index.d.ts

Anyways, I'll update this to an import to be a bit more explicit.

Reply:
Got it, ok sweet thanks @alukach